pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -389,6 +395,10 @@ spec:
             emptyDir: {}
           - name: dummyfile
             emptyDir: {}
+          - name: tmp
+            hostPath:
+              path: /tmp
```

Comment:
Can we check once if the hostpath is /tmp for the cstor-pool containers as well? Node's /tmp might not be persistent on reboots.